modules/installation-azure-limits.adoc

```diff
@@ -184,6 +184,17 @@ ifndef::ash[]
 endif::ash[]
 |The internal load balancer, each of the three control plane machines, and each
 of the three worker machines each use a private IP address.
+
+ifndef::ash[]
+|Spot VMs
```

Comment:
I think "Spot VM vCPU" should be more accurate

Reply:
minimum spot vm vCPU = worker node number *2

Reply:
Commend addressed.